frontend/src/components/EncryptedChat.js

```diff
@@ -20,6 +20,7 @@ import { saveAsJson } from '../utils/saveFile';
 import { AuditPGPDialog } from './Dialogs';
 import RobotAvatar from './Robots/RobotAvatar';
 import { systemClient } from '../services/System';
+import { websocketClient } from '../services/Websocket';
```

Comment:
Legacy unused import `import ReconnectingWebSocket from 'reconnecting-websocket';`

Reply:
no no, this is the new `websocketClient` : https://github.com/Reckless-Satoshi/robosats/pull/277/files#diff-6d4caf7a275f6bc3f26d740379aa63a0d2cddbcbccf44e6611f909af0e68793cR14

Reply:
Haha, sorry for the confusion, I could not leave a comment on a line that had not changed. I was pointing out that there is an unused import a few lines before the one I commented.